src/components/SeparatorList/index.test.tsx

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 
 import SeparatorList from '.';
 
 describe('Section component', () => {
-  it('matches snapshot for string items', () => {
-    const component = shallow(
+  it('renders string items', () => {
+    render(
       <SeparatorList
         separator="🐍"
         items={['why', 'did', 'it', 'have', 'to', 'be', 'snakes?']}
       />
     );
-    expect(component).toMatchSnapshot();
+    const items = screen.getAllByRole('listitem');
+    expect(items.length).toEqual(7);
```

Comment:
looks good but out of curiosity is length the best way to check this? What if some are undefined?

Reply:
Here's a snippet of code I found through googling: 

```
it("should render list of 5 fruits", () => {
  render(<FruitList />)
  const list = screen.getByRole("list", {
    name: /fruits/i,
  })
  const { getAllByRole } = within(list)
  const items = getAllByRole("listitem")
  expect(items.length).toBe(5)
})
```

Reply:
ohh this is interesting, I'll check it out

Reply:
i changed it so we can discuss it here.

i'm not sure where you stand on this @reywright, feels like the snippet, and the blog (i went into it to look at the whole thing) does the same thing but more verbose.

this looks like a more robust lookup of elements, or what am i missing?

Reply:
I guess just to state the outcome: you're adding specific text in the test. `'why', 'did', 'it'`, etc. Currently we're just making an assertion on how many list items there are. I think we would want assertions to check if the strings we're passing in are actually there.